### ISSUE TITLE
Open new tabs

### ISSUE THREAD

**Issue by vtmdsm27** [usagov/usa.gov#5]
Reminds me of the award-winning gov.uk. The site pages use plain language, and are uncluttered with extemporaneous info.Nice simple navigation - simple is best, and very hard to do.  

Pages have a plethora of links. But so far, the external links I've tried take me to a new site, completely away from the site page that has more links for me to explore.

Would it be useful to the viewer to have external page links open new tabs/windows rather than navigate away from usa.gov?  I'd like other thoughts on this.

**Comment by jrears** [comment]
I agree

**Comment by smileytech** [comment]
Thanks for the feedback!  We definitely looked at Gov.uk when designing the new site.  
We did discuss opening external links in new tabs/windows, but ultimately decided against it.  This is for a few reasons:
- Users may be confused by opening links automatically in new windows.  This was especially a concern for users who may have certain accessibility requirements.
- Related to the item above, opening links in new tabs inherently breaks the back button.  Users are used to being able to click the back button to go back to the previous page they were looking at, and opening links in a new window would preclude this.
- Users who would like to open links in a new window can always use their browser's right-click menu to do so.
- We worried that on mobile devices, opening new tabs might be a less than ideal user experience.
